src/xrCore/Animation/Bone.hpp

```diff
@@ -465,60 +455,61 @@ class XRCORE_API CBoneData : public IBoneData
     using ChildFacesVec = xr_vector<FacesVec>;
     ChildFacesVec child_faces; // shared
 
-    CBoneData(u16 ID) : SelfID(ID) { VERIFY(SelfID != BI_NONE); }
-    virtual ~CBoneData() {}
+    explicit CBoneData(u16 ID) : SelfID(ID) { VERIFY(SelfID != BI_NONE); }
+    virtual ~CBoneData() = default;
```

Comment:
I'm suspicious about this. Some code may be dependent on empty constructor. (because empty constructor doesn't default initialize variables)

Reply:
The default constructor doesn't implicitly initialize fields also. The only difference is that `CBoneData` may be trivial to construct now.

Reply:
This is a destructor, not a constructor. I'm not sure what are you trying to say.

Reply:
Ahh, nevermind then.